### PR TITLE
chore: update component tutorials README and vite.config.json files.

### PR DESCRIPTION
**Pull request by sagewall**
This PR updates the README files for the two components tutorials to add a brief getting started section for Vite.  It also adds a `vite.config.json` file to the tutorials.

### CODE REVIEW

File: component-samples/charts-components/tutorials/charts-components-with-map-components-solution/vite.config.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+import { defineConfig } from "vite";
+
+export default defineConfig({
+  plugins: [],
+  server: {
+    open: true,
+  },
+  build: {
+    outDir: "dist",
```
Comment (andygup) [inline component-samples/charts-components/tutorials/charts-components-with-map-components-solution/vite.config.js:24]: I also suggested adding some equivalent to this:

```
  build: {
    chunkSizeWarningLimit: 1500
  },
```

File: component-samples/map-components/tutorials/create-a-web-app-using-components-initial/vite.config.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { defineConfig } from "vite";
+
+export default defineConfig({
+  plugins: [],
+  server: {
+    open: true,
```
Comment (andygup) [inline component-samples/map-components/tutorials/create-a-web-app-using-components-initial/vite.config.js:21]: Ditto on chunk size warning settings.

File: component-samples/map-components/tutorials/create-a-web-app-using-components-solution/vite.config.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { defineConfig } from "vite";
+
+export default defineConfig({
+  plugins: [],
+  server: {
```
Comment (andygup) [inline component-samples/map-components/tutorials/create-a-web-app-using-components-solution/vite.config.js:20]: Ditto.

File: component-samples/charts-components/tutorials/charts-components-with-map-components-initial/vite.config.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { defineConfig } from "vite";
+
+export default defineConfig({
+  plugins: [],
+  server: {
```
Comment (andygup) [inline component-samples/charts-components/tutorials/charts-components-with-map-components-initial/vite.config.js:20]: Ditto on chunk size warning.

**Review by andygup — APPROVED** [review]
Looks good, just a suggestion to add the chunk size warning config.

**Comment by sagewall** [comment]
Oops I just copied the one in the vite sample without looking closely.  I'll update 😄

**Pull request merged**